mappings/net/minecraft/world/gen/foliage/SpruceFoliagePlacer.mapping

```diff
@@ -1 +1,4 @@
 CLASS net/minecraft/class_4650 net/minecraft/world/gen/foliage/SpruceFoliagePlacer
+	METHOD <init> (II)V
+		ARG 1 radius
+		ARG 2 radiusRandomness
```

Comment:
`-Random` or `-Randomness`? The others seem to use `-Random`.

Reply:
I thought that "random" was only for the Random class as it is used all over the code and could cause some real great confusion. I am ok with renaming just to something that suites it more as it is the radius's randomness.

Reply:
`radius_random` is the serialized name for the field (see `FoliagePlacer.serialize`), so `radiusRandom` is correct.

Reply:
upstream shows as follows

```java
    public <T> SpruceFoliagePlacer(Dynamic<T> dynamic) {
        this(dynamic.get("radius").asInt(0), dynamic.get("radius_random").asInt(0));
```

Reply:
you can see that it is called **radius_random**

Reply:
Yeah, it is, so I think they all should be `radiusRandom` for consistency.

Reply:
I agree is there anything that would break as a re-naming ?

Reply:
No, since the field is already `radiusRandom`, and I think the `radiusRandomness` names are only in parameter names added in this PR?

Reply:
It was added by me in this for Spruce I have to look into the other classes

Reply:
I think we should stick with `radiusRandom`. `radiusRandomness` is needlessly verbose.

Reply:
I agree and if you look I am the one that made the changes on all of them. so that will not break anything.I will change and post hash

Reply:
Fixed https://github.com/FabricMC/yarn/pull/1218/commits/ab8103072df0636dafd181c29df4a71b2ebce62c